### ISSUE TITLE
intro: clarify entry point into workflows

### ISSUE THREAD

**Issue by simonpcouch** [tidymodels/workshops#427]
The current introduction to workflows (https://github.com/tidymodels/workshops/issues/418) includes this slide:

![](https://private-user-images.githubusercontent.com/35748691/485171933-9b0b289b-efb6-421b-8672-5a2769b8f795.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTY5OTkxMDAsIm5iZiI6MTc1Njk5ODgwMCwicGF0aCI6Ii8zNTc0ODY5MS80ODUxNzE5MzMtOWIwYjI4OWItZWZiNi00MjFiLTg2NzItNWEyNzY5YjhmNzk1LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTA5MDQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwOTA0VDE1MTMyMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE5NmEzY2E2OWQxMTM3OGI4NjdjNTc0MGYwMjdkMDYyMTdiYjEwOGM3YjkyMWY4NGNiYjA2YmYwYjhkM2NkMmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.r6uoGfZ4JgK75x3ibFYGIyblS3gSjcml5HhN9DKdbGI)

Let's:

* Replace PCA with Centering + Scaling as an example of preprocessing that requires estimation
* Undo the delineation of "Data" and "Predictors"
* Add a speaker note that the explanation of data leakage here uses the example of preprocessing _on the whole source data set_ rather than _on the whole training set_. So, the data leakage risk is "the values from the test set affect the mean and standard deviations used in the preprocessing." That discussion doesn't need to mention metrics, as they haven't been introduced yet.
